public/components/map_container/map_container.tsx

```diff
@@ -46,10 +41,7 @@ interface MapContainerProps {
   mapState: MapState;
   mapConfig: ConfigSchema;
   isReadOnlyMode: boolean;
```

Comment:
is it possible to use dashboardProps check instead of keeping isReadOnlyMode?

Reply:
I prefer to keep `isReadOnlyMode` prop, as the name means, we shouldn’t restrict the use case for dashboard mode or other type of view mode in the future, we already had the discussion when we defined the `isReadOnlyMode` name

Reply:
Agree, before we needed 1 parameter to identify whether we need to render for Map or  inside dashboards. Now, you can do that from dashboardProps.  for ex: if dashboardProps is not undefined,  you can infer readonly .